src/borg/archiver.py

```diff
@@ -2412,10 +2412,31 @@ def do_break_lock(self, args, repository):
                 # susan is a nice person
                 # include susans home
                 + /home/susan
+                # also back up this exact file
+                + pf:/home/bobby/specialfile.txt
                 # don't backup the other home directories
                 - /home/*
                 # don't even look in /proc
-                ! /proc\n\n''')
+                ! /proc
+
+            As another example, you could do the same thing in either of these ways::
```

Comment:
"same" might get misunderstood as a backreference to the previous example (lines before 2420).

maybe rather say:

Here is another example that shows how recursion roots and excludes can be given either directly on the command line or in a patternfile.

(with that, you could drop the comments from 2424 and 2426)